rct229/utils/std_comparisons.py

```diff
@@ -1,4 +1,5 @@
 DEFAULT_PERCENT_TOLERANCE = 0.05
+AHJ_RA_COMPARE = False
```

Comment:
Add comments to this line to explain what is AHJ_RA_COMPARE, and the meaning for True and False.